bridge/server/services/data-service.ts

```diff
@@ -207,9 +207,15 @@ export class DataService {
     return validRegistrations;
   }
 
+  public async getIsUniformRegistrationControlPlane(integrationId: string): Promise<boolean> {
+    const response = await this.apiService.getUniformRegistrations(integrationId);
+    const uniformRegistration = response.data.shift();
+    return uniformRegistration?.metadata.location === 'control-plane';
```

Comment:
We have this string on several places in the code. Should we put it in a model, so that if it changes somewhere in the future, the change can be done on one place in the code?